lib/api_client/execute_request.js

```diff
@@ -98,8 +106,9 @@ function execute_request(method, params, auth, api_url, callback, options = {})
 
         if (result.error) {
           deferred.reject(Object.assign({
-            request_options,
-            query_params
+            request_options: sanitized_request_options,
+            query_params,
+            hide_sensitive
```

Comment:
I wouldn't return the value for `hide_sensitive` in the error. The behaviour we implemented here is by default turned off, so the developer needs to explicitly set it to `true`.

Think about it this way: if we return `hide_sensitive`, why not return the whole config?